adafruit_shell.py

```diff
@@ -369,6 +373,31 @@ def copy(self, source, destination):
                     destination += os.sep + os.path.basename(source)
                 shutil.copy(source, destination)
 
+    def chmod(self, location, mode):
+        """
+        Change the permissions of a file or directory
+        """
+        location = self.path(location)
+        if os.path.exists(location):
+            os.chmod(location, mode)
+
+    def chown(self, location, user, group=None, recursive=False):
```

Comment:
May want to match kwargs with the Python `chown`, https://docs.python.org/3.11/library/os.html#os.chown

Reply:
Good idea. Thanks.

Reply:
It looks like there's nothing additional for the shutil.chown function: https://docs.python.org/3/library/shutil.html#shutil.chown.